.build/BuildToolkit.ps1

```diff
@@ -27,7 +27,6 @@ $NugetPackageArtifacts = "$ArtifactsDir\Packages";
 . "$PSScriptRoot\Output.ps1";
 
 # Define Tools
-$global:MSBuildCli = "msbuild.exe";
 $global:DotNetCli = "dotnet.exe";
 $global:NugetCli = "nuget.exe";
```

Comment:
Do we still need `NugetCli`?

Reply:
yes, eg for install tools (which are not migrated to `dotnet tool`)